components/App.js

```diff
@@ -1,24 +1,75 @@
-import React from 'react'
+// import React, {useEffect, useState } from 'react'
+import React, { Component } from 'react'
 import { NavigationContainer } from '@react-navigation/native'
 import { createNativeStackNavigator } from '@react-navigation/native-stack'
 import HomeScreen from './HomeScreen'
-import LogFlightScreen from './LogFlightScreen'
+import FlightForm from './FlightForm'
 import FlightHistoryScreen from './FlightHistoryScreen'
 import FlightDetailsScreen from './FlightDetailsScreen'
+import { getUser, getUserFlights } from '../ApiCalls'
+
 
 const Stack = createNativeStackNavigator()
 
-function AppNavigator() {
-  return (
-    <NavigationContainer>
-      <Stack.Navigator initialRouteName="Home">
-        <Stack.Screen name="Home" component={HomeScreen} />
-        <Stack.Screen name="LogFlight" component={LogFlightScreen} />
-        <Stack.Screen name="FlightHistory" component={FlightHistoryScreen} />
-        <Stack.Screen name="FlightDetails" component={FlightDetailsScreen} />
-      </Stack.Navigator>
-    </NavigationContainer>
-  )
+class AppNavigator extends Component  {
+  constructor() {
+    super()
+    this.state = {
+      user: {},
+      userFlights: [], 
+      error: ''
```

Comment:
This is great for now, we can add a loading property to state at a later point for a better user experience.